components/admin_console/workspace-optimization/dashboard.tsx

```diff
@@ -1,8 +1,167 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import React from 'react';
+import classNames from 'classnames';
+import React, {useEffect, useState} from 'react';
+import styled from 'styled-components';
 
-const WorkspaceOptimizationDashboard = () => <div>{'Workspace Optimization Dashboard'}</div>;
+import {testSiteURL} from '../../../actions/admin_actions';
+
+import {Props} from '../admin_console';
+
+type DataModel = {
+    [key: string]: {
+        title: string;
+        description: string;
+        items: ItemModel[];
```

Comment:
Do we need a field for the accordion icon or is that done via CSS?

Reply:
I would assume we do that based on the items: If the length of the array is > 0 show an item. This brings up another interesting question though: Do we need to show the topic when there is nothing to do?

Reply:
Good question you brought up. My question was more about these:

<img width="1123" alt="Screen Shot 2022-01-18 at 11 50 30 AM" src="https://user-images.githubusercontent.com/1149597/149981879-1d0fa570-c4ef-45a7-88e4-c92dbf069bb7.png">

Reply:
oh no ... that should be defined by the dataModel (at least the icon)